src/main/kotlin/graphics/scenery/controls/behaviours/VRPress.kt

```diff
@@ -108,15 +146,29 @@ open class VRPress(
     }
 }
 
+/**
+ * Top level class for press able nodes in VR interaction.
+ *
+ *  See [SimplePressable] for functionality that was previously implemented under this name.
+ */
+sealed class Pressable
```

Comment:
For cleanliness, I think it'd be good to move this to it's own file.

Reply:
And actually also for consistency ^^

Reply:
For consistency though it should stay in this file because the *-able is always in the same file as the behavior.